### DISCUSSION TITLE
new release 0.14.2

### DISCUSSION THREAD

**Discussion by xalagu** [jianyuan/terraform-provider-sentry#546] [Q&A] [answered]
Hello @jianyuan 

Plans for a new release ?
I ran into a already fixed problem (https://github.com/jianyuan/terraform-provider-sentry/issues/516) but not released yet !

**Comment by jianyuan** [comment, marked as answer]
It's going out today! https://github.com/jianyuan/terraform-provider-sentry/releases/tag/v0.14.2
